src/components/atoms/ClickableIcon.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from "styled-components";
+import {Link} from "gatsby";
+
+const Icon = styled.img`
+    height: 25px;
+`
+
+function IconsPanel({link, img, href, alt}) {
+    return href? (
+        <a target = "blank" href = {href} >
```

Comment:
Is this to handle the email icon without a link? 
Apologies for not including the link for this! You can use `info@inthefight.org`

Reply:
nah this was an option if you want to link to an outside site you use the href prop, and if you want to navigate in your site you use a link prop

okay I will add it now

Reply:
Got it! Thanks for the clarification